ui/components/multichain/token-list-item/price/percentage-change/percentage-change.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Box, Text } from '../../../../component-library';
+import {
+  Display,
+  FontWeight,
+  TextColor,
+  TextVariant,
+} from '../../../../../helpers/constants/design-system';
+import { isValidAmount } from '../../../../../../app/scripts/lib/util';
+
+export const PercentageChange = ({
+  value,
+  address,
+}: {
+  value: number | null | undefined;
+  address: `0x${string}`;
+}) => {
+  let color = TextColor.textDefault;
+
+  if (isValidAmount(value)) {
+    if ((value as number) >= 0) {
```

Comment:
Non-blocking idea: in the very rare case that the percentage change is `0`, we should maybe use a neutral color.